backend/qemu.pm

```diff
@@ -490,6 +490,7 @@ sub start_qemu {
     bmwqemu::save_vars();                 # update variables
 
     mkpath($basedir);
+    runcmd('/usr/bin/chattr', '+C', $basedir);
```

Comment:
chattr can be silenced by adding "-f".
The return value (1 on error, 0 otherwise) is currently not evaluated anyway.

@coolo: Do you have any pointers regarding the apparmor profile?